internal/featuretests/kubernetes.go

```diff
@@ -124,3 +127,19 @@ func secretdata(cert, key string) map[string][]byte {
 		v1.TLSPrivateKeyKey: []byte(key),
 	}
 }
+
+func meta(name string) metav1.ObjectMeta {
+	v := strings.SplitN(name, "/", 2)
+	switch len(v) {
+	case 1:
+		// no prefix
+		return metav1.ObjectMeta{
+			Name: v[0],
```

Comment:
I don't think this is safe to do, meta("foo") produces 
```
metav1.ObjectMeta{
  Namespace: "",
  Name: "foo",
}
```
I don't think we should let fixtures like this be created.

Reply:
OK, I can just add a panic in that case ...